lib/git-shell-out-strategy.js

```diff
@@ -595,6 +505,10 @@ export default class GitShellOutStrategy {
     return output.trim().split(LINE_ENDING_REGEX);
   }
 
+  async describeHead() {
+    return (await this.exec(['describe', '--contains', '--all', '--always', 'HEAD'])).trim();
+  }
```

Comment:
:+1: Cool, I was wondering if you'd preserved the `describe`-for-detached-HEADs bit.